python_modules/dagster-graphql/dagster_graphql/implementation/execution/backfill.py

```diff
@@ -114,11 +114,14 @@ def create_and_launch_partition_backfill(
     partitions_by_assets = backfill_params.get("partitionsByAssets")
 
     if (
-        asset_selection or backfill_params.get("selector") or backfill_params.get("partitionNames")
+        asset_selection
+        or backfill_params.get("selector")
+        or backfill_params.get("partitionNames")
+        or backfill_params.get("allPartitions")
     ) and partitions_by_assets:
         raise DagsterInvariantViolationError(
-            "partitions_by_assets cannot be used together with asset_selection, selector, or"
-            " partitionNames"
+            "partitionsByAssets cannot be used together with assetSelection, selector, or"
```

Comment:
At the very least, I feel like the capitalization should be consistent; the camel-casing is aligned with the GraphQL API names.